src/twisted/test/test_failure.py

```diff
@@ -22,7 +22,7 @@
 from twisted.trial.unittest import SynchronousTestCase
 
 
-from cython_test_exception_raiser import raiser
+from cython_test_exception_raiser import raiser  # type: ignore[import]
```

Comment:
This is a Twisted-organization module, perhaps we should open a ticket to annotate it so we can remove this `ignore` in the future